src/main/java/buildcraftAdditions/proxy/ClientProxy.java

```diff
@@ -102,4 +103,8 @@ public int addArmor(String name) {
 	public void addListeners() {
 		FMLCommonHandler.instance().bus().register(new KeyListener());
 	}
+
+	public static GuiScreen getOpenGui() {
```

Comment:
GuiScreen doesn't exist on the server so not in the commonproxy. I just put this here as this was the best place I could thing of to place the method